authors/apps/articles/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from .views import ListCreateArticleAPIView, RetrieveUpdateArticleAPIView
+
+app_name = 'articles'
+
+urlpatterns = [
+    path('articles/', ListCreateArticleAPIView.as_view(), name='article'),
+    path('articles/<slug>', RetrieveUpdateArticleAPIView.as_view(),
```

Comment:
Please include a `/` a the end of the URL pattern here.

Reply:
Already done